### DISCUSSION TITLE
template with backticks?

### DISCUSSION THREAD

**Discussion by Yellow-Subroutine** [gsantner/markor#2009] [General] [answered]
Hi,

I am having problems using templates that start with backticks (I want one for mermaid) - they seem to get stripped out when inserting the template.

Is there a trick to do that?

**Comment by harshad1** [comment, marked as answer]
backslash to escape.

i.e. 

```
This is a template with backticks \` text in backticks \`
```

**Comment by harshad1** [comment]
Note that strings delimited by backticks are interpreted as dates in the java simpledate format

i.e. 

```
Today is `yyyy-mm-dd`
```

will print _Today is 2023-04-04_
